sparta/sparta/ports/SyncPort.hpp

```diff
@@ -194,6 +194,37 @@ namespace sparta
             return sync_in_port_->getRawReady_();
         }
 
+        /*! \brief Determine if this DataOutPort has any connected
+         *        DataInPort where the data is to be delivered on the
+         *        given cycle.
```

Comment:
Can you fix the comments to reflect this is a SyncPort?

Reply:
All comments fixed in next commit.